src/Silex/AppArgumentValueResolver.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the Silex framework.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Silex;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Controller\ArgumentValueResolverInterface;
+use Symfony\Component\HttpKernel\ControllerMetadata\ArgumentMetadata;
+
+/**
+ * HttpKernel Argument Resolver for Silex.
+ *
+ * @author Romain Neutron <imprec@gmail.com>
+ */
+class AppArgumentValueResolver implements ArgumentValueResolverInterface
```

Comment:
Is this class required for event a new event dispatcher plugins?

Reply:
No, it's only used for testing of Silex.